src/main/scala/viper/server/ViperServer.scala

```diff
@@ -6,21 +6,51 @@
 
 package viper.server
 
-import viper.server.frontends.http.ViperHttpServer
+import java.io.IOException
+import java.net.Socket
+
+import org.eclipse.lsp4j.jsonrpc.Launcher
+import viper.server.frontends.lsp.{Coordinator, CustomReceiver, IdeLanguageClient}
 
 object ViperServerRunner {

```

Comment:
Are you sure this supports backwards compatibility? If we merge the PR, the existing clients will use this class to start ViperServer, but I think this is the class that starts the LSP backend, right?

Reply:
Yes, I must have mixed up the two runner objects.